### DISCUSSION TITLE
Welcome to Tafe230132 Discussions!

### DISCUSSION THREAD

**Discussion by MatthewJRobertson94** [MatthewJRobertson94/Tafe230132#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vincentx1908** [comment]
It's great to be your teammate! I think we can make it as long as we work together!
